### DISCUSSION TITLE
Cropped plots in VSCode with Jupyter notebooks

### DISCUSSION THREAD

**Discussion by DavidSiretMarques** [microsoft/vscode-discussions#635] [Extension Development QnA] [answered]
I'm having a problem in VSCode when using Jupyter Notebooks with python and matplotlib, the 3d plots appear cropped.

I'd like to raise an issue here but I don't really know where to post it.

The full problem can be seen in [this stackoverflow question](https://stackoverflow.com/questions/76358689/uncrop-3d-plots-in-jupyter-notebook-with-vscode)

**Comment by gjsjohnmurray** [comment]
The right place for this is https://github.com/microsoft/vscode/issues

**Reply by DavidSiretMarques** [reply]
Thanks!

**Comment by gjsjohnmurray** [comment, marked as answer]
Or maybe start at https://github.com/microsoft/vscode-jupyter/issues in case it is extension specific.

**Reply by DavidSiretMarques** [reply]
That's the point, I'm not sure. I know in jupyter lab it doesn't happen, but I'm not sure how to check if it's extension specific...

**Comment by gjsjohnmurray** [comment]
I recommend you start at the extension repo. If they determine that the problem is in core VS Code they can raise it there.

**Reply by DavidSiretMarques** [reply]
Thanks for all the info
